docs/internal/demo-server-guide.md

```diff
@@ -0,0 +1,77 @@
+# mFlow Demo Server build and install using Docker
+
+## Prep
+
+- Add token to githubtoken.txt (for downloading expression-evaluator)
+  - requires token that has read access to packages on server repo
+- Edit `dockerise.sh` and update `SERVER_BRANCH`, `WEB_APP_BRANCH` and `IMAGE_NAME`
+
+## Build
+
+`cd docker`  
+`./dockerise.sh`
+
+Should take a while to run, will build a local image
+
+## Test locally
+
+- Edit `run.sh`
+  - include correct `SMTP_PASSWORD`(from Bitwarden or your personal .env file)
+  - Change test build to name of image created
+- Run it: `./run.sh`
+
+## Upload to Docker hub
+
+- Login to docker hub (web site) to see the images in there:
+  - msupplyfoundation - password in Bitwarden (Shared-Admin)
+- List local images to get the “image ID” and "tag name"  
+  `docker image ls`
+- Create tag:  
+  `docker tag <image_id> /<docker-repo>:<tag-name>`
```

Comment:
```suggestion
  `docker tag <image_id> <organisation>/<docker-repo>:<tag-name>`
```

Reply:
Oh, this was fixed on the other PR -- I branched from that, should have made that the base here sorry.